feedback/pretalx_client.py

```
@@ -0,0 +1,1050 @@
+        req = urllib.request.Request(url, method=method)
+        req.add_header("Authorization", f"Token {self.apikey}")
+        req.add_header("Accept", "application/json")
+        if self.version:
+            req.add_header("Pretalx-Version", self.version)
+            
+        if data is not None:
+            req.add_header("Content-Type", "application/json")
+            req.data = json.dumps(data).encode("utf-8")
+            
+        try:
+            with urllib.request.urlopen(req) as response:
```

Ruff (0.16.3)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)